internal/consensus/common_test.go

```diff
@@ -429,22 +429,36 @@ func subscribeToVoterBuffered(cs *State, addr []byte) <-chan cmtpubsub.Message {
 	return ch
 }
 
+// -------------------------------------------------------------------------------
+// application
+
+func newAppWithInfo(t *testing.T) (*kvstore.Application, []byte, *mempl.LanesInfo) {
```

Comment:
I would return the whole response here so the tests can reuse what they need. We might have a case in the future where we need more fields.

Reply:
Done.